libs/analytics/src/googleAnalytics/initGoogleAnalytics.ts

```diff
@@ -0,0 +1,38 @@
+import { CowAnalyticsGoogle } from './CowAnalyticsGoogle'
+
+import { CowAnalytics } from '../CowAnalytics'
+
+export const GOOGLE_ANALYTICS_CLIENT_ID_STORAGE_KEY = 'ga_client_id'
+
+export function initCowAnalyticsGoogle(): CowAnalytics {
+  // Load stored client id
+  let storedClientId: string | null = null
+  if (typeof window !== 'undefined') {
+    storedClientId = window.localStorage.getItem(GOOGLE_ANALYTICS_CLIENT_ID_STORAGE_KEY)
+  }
+
+  // Init analytics
+  const googleAnalyticsId = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS || process.env.REACT_APP_GOOGLE_ANALYTICS_ID
```

Comment:
I take we look for both to accept both app types. Is there ever a chance this might cause a conflict?
Would it work if we set a single one for both?

Reply:
> I take we look for both to accept both app types. Is there ever a chance this might cause a conflict?

No, unless you define both. In fact they need the same ID. What I can do is to make this a parameter of the init, so each app decides. I just left it like this to avoid a bit repetition. Happy to move it out to the apps if you prefer